### ISSUE TITLE
Screen reader does not announce the name for the 'Send' button.

### ISSUE THREAD

**Issue by msftedad** [FredrikOseberg/react-chatbot-kit#197]
**Title:** Screen reader does not announce the name for the 'Send' button.

**Test Environment:** 
OS: Windows 11 Version 24H2 (OS Build 26120.1542)
Browser: Microsoft New Edge (Version 128.0.2739.42) (Official build) (64-bit)
AT: Narrator

**Repro steps:**

1. Turn on Narrator by pressing 'Ctrl + Windows + Enter' keys.
2. Open URL [Hello from React-chatbot-kit | React-chatbot-kit (fredrikoseberg.github.io)](https://fredrikoseberg.github.io/react-chatbot-kit-docs/) in Edge browser.
3. 'React-chatbot-kit' page will open.
4. Press tab key to reach 'Chatbot Send' button.
5. Press tab key to reach Send' button in 'Write your message' edit field and observe whether the 'Send' button has a Name or not.

**Actual Result:** Screen reader is announcing just as 'button' for the 'Send' button. 

https://github.com/user-attachments/assets/7ba09f06-ef45-4268-9cfd-36ba79c93f8a

![Descriptive name is not provided for 'Send' button  (1)](https://github.com/user-attachments/assets/6afc157e-129a-4b1b-acec-050085167f59)

**Expected Result:** Screen reader should announce proper name as 'Send' button.

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?
